opencti-platform/opencti-front/src/private/components/threats/threat_actors_individual/ThreatActorIndividualDetails.tsx

```diff
@@ -152,17 +179,23 @@ const ThreatActorIndividualDetails: FunctionComponent<ThreatActorIndividualDetai
               )}
             </Grid>
             <Grid item={true} xs={6}>
-              <Typography variant="h3" gutterBottom={true}>
-                {t('Threat actor individual types')}
-              </Typography>
-              {data.threat_actor_types
-                && data.threat_actor_types.map((threatActorIndividualType) => (
-                  <Chip
-                    key={threatActorIndividualType}
-                    classes={{ root: classes.chip }}
-                    label={threatActorIndividualType}
-                  />
-                ))}
+              <FieldOrEmpty source={data.images}>
+                <Carousel
+                  height='150px'
+                  className={classes.carousel}
+                  animation='fade'
+                >
+                  {data.images && data.images.map((file) => (
+                    <Tooltip title={file ? file.description : ''} key={file ? file.id : ''} placement='right'>
```

Comment:
why do we have to check that the file is defined ? Is it because we declared : 
`x_opencti_files(prefixMimeType: String): [OpenCtiFile]`
instead of :
`x_opencti_files(prefixMimeType: String): [OpenCtiFile!]`

What is the best practice for this case @Kedae ?

Reply:
That's not a matter of best pratice that's a matter of what we want : does x_opencti_files = null means something ? does x_opencti_files = [null] means something ?

Reply:
in which cases does [null] means something ?

Reply:
I honestly never found any case where this is useful... Just discussed it with Sarah, she will update it accordingly.